src/Google/RestApi.php

```diff
@@ -278,12 +278,10 @@ protected function logRetryRequest(
         ?ResponseInterface $response = null
     ): void {
         if ($this->logger !== null) {
-            $headersForLog = array_map(function ($item, $key) {
-                if (strtolower($key) === 'authorization') {
-                    return '*****';
-                }
-                return $item;
-            }, $request->getHeaders(), array_keys($request->getHeaders()));
+            $headersForLog = $request->getHeaders();
+            if (array_key_exists('Authorization', $headersForLog)) {
```

Comment:
to že se to převádělo na malé písmena mělo bych řekl nějaký důvod a nechal bych to tu